incubator/java-spring-boot2/image/project/Dockerfile

```diff
@@ -1,23 +1,50 @@
-FROM {{.stack.image.namespace}}/{{.stack.id}}:{{.stack.semver.major}}.{{.stack.semver.minor}} as compile
+FROM maven:3.6-ibmjava-8 as compile
```

Comment:
What started happening: The FROM line was not obeying the registry (not just the namespace, but the registry). We tripped on this with the kabanero stack, where builds would fail trying to retrieve the as-yet-unpublished stack image. I started to think about how this would actually work in different build environments. You know I pioneered the whole template thing to reuse the stack image, but I started to really question feasibility because of issues with whether or not the registry (not just the namespace) needed to be included.

Reply:
Oh right we are referencing the image that is not yet published.. Will it still not work with newe --image-registry flag added to stack package/validate?

Reply:
I'm looking @ what happens when stack maintainers make a stack, and how that goes in to stack-config.yaml, and then how that named stack transitions across into the build environment...

Reply:
I moved that from image to a value in stack.yaml